boards/PDM/Src/App/states/App_FaultState.c

```diff
@@ -0,0 +1,106 @@
+
+#include "states/App_AllStates.h"
+#include "states/App_DriveState.h"
+#include "App_SharedMacros.h"
+
+static void FaultStateRunOnEntry(struct StateMachine *const state_machine)
+{
+    struct PdmWorld *const world = App_SharedStateMachine_GetWorld(state_machine);
+}
+
+bool Faults(
+    struct Efuse *         efuse1,
+    struct Efuse *         efuse2,
+    struct Efuse *         efuse3,
+    struct Efuse *         efuse4,
+    struct RailMonitoring *rail_monitor)
+{
+    int num_fixed = 0;
+    if (App_Efuse_FaultProcedure_Channel0(efuse1, 3) == 0)
+    {
+        // AIR
+        App_CanTx_PDM_Efuse_Fault_Checks_AIR_Set(false);
+        num_fixed++;
+    }
+    if (App_Efuse_FaultProcedure_Channel0(efuse2, 0) == 0)
+    {
+        // EMETER
+        App_CanTx_PDM_Efuse_Fault_Checks_EMETER_Set(false);
+        num_fixed++;
+    }
+    if (App_Efuse_FaultProcedure_Channel0(efuse3, 0) == 0)
+    {
+        // LEFT INVERTER
+        App_CanTx_PDM_Efuse_Fault_Checks_LEFT_INVERTER_Set(false);
+        num_fixed++;
+    }
```

Comment:
What are the conditions for PDM exiting fault state? These will probably fail because they're disabled, and so CurrentLowCheck will trip